python/ray/job_config.py

```diff
@@ -52,24 +54,37 @@ def __init__(self,
             f"{type(code_search_path)}"
         self.runtime_env = runtime_env or dict()
         self.client_job = client_job
+        self.ray_namespace = ray_namespace
```

Comment:
any particular reason to call this `ray_namespace` and not just `namespace`? Seems redundant

Reply:
Just consistency with the protobuf. On the protobuf and c++ side we can't call it `namespace` because it's a reserved keyword.